### ISSUE TITLE
Freeswitch pkgconfig file has wrong paths

### ISSUE THREAD

**Issue by neheb** [openwrt/telephony#561]
```
libdir=/usr/lib
includedir=/usr/include/freeswitch
```

should be

```
libdir=${prefix}/lib
includedir=${prefix}/include/freeswitch
```

**Comment by micmac1** [comment]
Hello Rosen,

This should be fixed in #562, please confirm.

Regards,
Seb

**Comment by micmac1** [comment]
Closing as fixed in trunk.
